pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -310,13 +309,14 @@ func (ca *cloudCIDRAllocator) updateCIDRAllocation(nodeName string) error {
 	}
 
 	cidrStrings := make([]string, 0)
-	var northInterfaces networkv1.NorthInterfacesAnnotation
-	var additionalNodeNetworks networkv1.MultiNetworkAnnotation
 
 	if len(instance.NetworkInterfaces) == 0 || (len(instance.NetworkInterfaces) == 1 && len(instance.NetworkInterfaces[0].AliasIpRanges) == 0) {
 		nodeutil.RecordNodeStatusChange(ca.recorder, node, "CIDRNotAvailable")
 		return fmt.Errorf("failed to allocate cidr: Node %v has no ranges from which CIDRs can be allocated", node.Name)
 	}
+
+	ca.setNetworkCondition(node)
```

Comment:
add a comment to explain what it does
```
// sets the v1.NodeNetworkUnavailable to False to indicate that the Network is ready
// if the condition is set to True it marks the node as unusable until routes are initialized in the cloud provider
```

Reply:
updated, I skipped the second sentence, since we are not setting True in this function